source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_cfg.py

```diff
@@ -175,6 +175,9 @@ class SimulationCfg:
     render_interval: int = 1
     """The number of physics simulation steps per rendering step. Default is 1."""
 
+    antialiasing_mode: str = "DLSS"
```

Comment:
When there are a specific list of options, it is better to use Literal so it is clear what options a user has

```suggestion
    antialiasing_mode: Literal["FXAA", "DLSS", "TAA", "RTXAA"] = "DLSS"
```